.github/workflows/build.yml

```diff
@@ -30,3 +30,13 @@ jobs:
             ${{ runner.os }}-gradle-v2-
 
       - run: ./gradlew --no-daemon build
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: patcher-189
+          path: versions/1.8.9-forge/*
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: patcher-1122
+          path: versions/1.12.2-forge/*
```

Comment:
This should be a single upload action which uses a globstar (`**`) as a path, see https://github.com/Polyfrost/OneConfig/blob/develop-v0/.github/workflows/build.yml#L51-L55

Reply:
> This should be a single upload action which uses a globstar (`**`) as a path, see https://github.com/Polyfrost/OneConfig/blob/develop-v0/.github/workflows/build.yml#L51-L55

think it's ready